bezsmertnyi-stanislav/Readme.MD

```diff
@@ -0,0 +1,68 @@
+Flashscore Live Scores
+======================
+
+## Introduction
+
```

Comment:
Notes on the introduction:
* The sentences don't form a coherent paragraph: each sentence pulls in its own direction and is not coordinated with the others. Sentence one describes the app. Sentence two is an imperative. Sentence three lists a couple of features (and is a fragment -- in English, sentences should always have a subject and a verb). Sentence four lists more features, but this time by directly addressing the reader.
* Your intro repeats the points you provided in the list of features. There's no need to duplicate this information; moreover, the intro should be as clear and brief as possible. In your case, you can leave the first sentence and it would be enough.